### DISCUSSION TITLE
能否一条命令配置编译？

### DISCUSSION THREAD

**Discussion by PeakRacing** [xmake-io/xmake#2917] [Q&A]
现在配置编译需要两条命令
xmake f --test=y
xmake
请问我想一条命令执行配置+编译直接有办法么？

**Comment by waruqi** [comment]
```console
xmake f --test=y; xmake
```

或者 xmake.lua 配置默认启用 test，只需要 `xmake`

**Comment by PeakRacing** [comment]
谢谢，我想通过这种方法控制编译哪个项目，所以会涉及修改，那我xmake f --test=y; xmake这样吧
